src/language/jump_es.properties

```diff
@@ -3008,3 +3008,4 @@ org.openjump.core.ui.plugin.tools.MergeSelectedPolygonsWithNeighbourPlugIn.merge
 org.openjump.core.ui.plugin.tools.MergeSelectedPolygonsWithNeighbourPlugIn.minimum-area=#T:minimum area
 org.openjump.core.ui.plugin.tools.MergeSelectedPolygonsWithNeighbourPlugIn.Skip-null-values=#T:Skip features with null attribute value
 jump.features-processed=elementos procesados
+org.openjump.core.ccordsys.AuxFile-srs=SRS se incluye en un archivo auxiliar
```

Comment:
this new key needs to be added to **all language files**
especially to the english version `src/language/jump.properties` with english translation